pkg/data/components/ParseAttributeAsJSON.yaml

```diff
@@ -0,0 +1,57 @@
+kind: ParseAttributeAsJSON
+name: Parse Attribute As JSON
+style: processor
+type: base
+status: development
+version: v0.0.1
+summary: Converts an Attribute's JSON string value into individual attributes
+description: |
+  Takes any attribute from a log or span and parses it as JSON into individual attributes
+tags:
+  - category:processor
+  - service:collector
+  - category:filter
+  - signal:OTelTraces
+  - signal:OTelLogs
+ports:
+  - name: Logs
+    direction: input
+    type: OTelLogs
+  - name: Logs
+    direction: output
+    type: OTelLogs
+  - name: Spans
+    direction: input
+    type: OTelTraces
+  - name: Spans
+    direction: output
+    type: OTelTraces
+properties:
+  - name: Attribute
+    type: string
+    default: "json_data"
+  - name: Signal
+    type: string
+    subtype: oneof(span, log)
+    default: "log"
+    validations:
+      - oneof(span, log)
```

Comment:
We can allow datapoint now too.

Reply:
I don't want to encourage that, maybe we can add that later? I don't see that as a common usecase at all